### ISSUE TITLE
slow

### ISSUE THREAD

**Issue by alienw4re** [selbstmord/accountnuke#2]
this fucking nuker  is so slow

**Comment by selbstmord** [comment]
cry about it

**Comment by alienw4re** [comment]
> cry about it

No, You. you fucking pathetic waste of sperm. how fucking useless are you? YOU cant even code an account nuker correctly you degenerate, stfu.

**Comment by selbstmord** [comment]
refer to the time i said "cry about it"

**Comment by AmpPy** [comment]
refer to the time he said "No, You. you fucking pathetic waste of sperm. how fucking useless are you? YOU cant even code an account nuker correctly you degenerate, stfu."

**Comment by selbstmord** [comment]
😹
